js/app.jsx

```diff
@@ -486,6 +486,26 @@ var Job = React.createClass({
       </ul>
     );
 
+    function slimTrace(str) {
+      var lines = [];
+      if (str) {
+        lines = str.split(/,|ÿ/)
```

Comment:
Not sure why, but the lines are delimited with this weird `ÿ` for `scalding.pipe.descriptions` at `jobhistory.stripe.com`